techmatters/flex-plugins
# CHI-1526: Search result styling tweaks

Primary reviewer: @mmythily 

## Description

* Small tweaks to the layout requested by Jana
* Fix some misaligned translations

### Checklist
- N/A New tests added
- N/A Feature flags added
- N/A Strings are localized

### Related Issues
Fixes CHI-1580

### Verification steps

* See ticket
* eyeball the case & contact search results
* test the 'see more' and 'see less' function
